src/util/modules.py

```diff
@@ -525,9 +544,18 @@ def force_install_module(cr, module, if_installed=None):
     )
 
     states = dict(cr.fetchall())
-    # auto_install modules...
     toinstall = [m for m in states if states[m] == "to install"]
-    if toinstall:
+
+    if module in toinstall:
+        _logger.info(
+            "force install of module %r (and its dependencies) because %s%s",
+            module,
+            reason,
+            " and modules {!r} are installed".format(if_installed) if if_installed else "",
```

Comment:
I found it a bit confusing:
```suggestion
            " and modules {!r} are already installed".format(if_installed) if if_installed else "",
```